src/Relude/Extra/Validation.hs

```diff
@@ -57,26 +75,17 @@ Failure ["Not correct"]
 >>> c *> d *> b
 Failure ["Not correct","Not correct either"]
 
->>> liftA2 (+) a b
+>>> (+) <$> a <*> b
```

Comment:
The fact that we don't implement `liftA2` doesn't mean that we cannot use it. I think that it's completely okay to use `liftA2` in this documentation.